srv/modules/runners/replace.py

```diff
@@ -52,23 +53,24 @@ def osd(*args, **kwargs):
 
     local = salt.client.LocalClient()
     host_osds = local.cmd('I@roles:storage', 'osd.list', tgt_type='compound')
+    assert isinstance(host_osds, list)
```

Comment:
`local.cmd('I@roles:storage', 'osd.list', tgt_type='compound')` returns a dict of host: list_of_osds